frontend/src/metabase/search/components/SearchSidebar/SearchSidebar.tsx

```diff
@@ -12,6 +12,7 @@ import { TypeFilter } from "metabase/search/components/filters/TypeFilter";
 import { PLUGIN_CONTENT_VERIFICATION } from "metabase/plugins";
 import { ToggleSidebarFilter } from "metabase/search/components/ToggleSidebarFilter";
 import { CreatedByFilter } from "metabase/search/components/filters/CreatedByFilter";
+import { SQLNativeQueryFilter } from "metabase/search/components/filters/SQLNativeQueryFilter";
```

Comment:
The filter should apply to native queries, not SQL queries only
```suggestion
import { NativeQueryFilter } from "metabase/search/components/filters/NativeQueryFilter";
```